pydantic/_internal/_generate_schema.py

```diff
@@ -1130,7 +1130,9 @@ def _namedtuple_schema(self, namedtuple_cls: Any, origin: Any) -> core_schema.Co
 
             arguments_schema = core_schema.arguments_schema(
                 [
-                    self._generate_parameter_schema(field_name, annotation)
+                    self._generate_parameter_schema(
+                        field_name, annotation, default=namedtuple_cls._field_defaults.get(field_name, Parameter.empty)
```

Comment:
Though this looks like a private attribute access, this `_field_defaults` attribute is explicitly present in the type stubs for `NamedTuple` — I think it's intended for public access, just has an underscore to prevent name collisions.

Reply:
https://github.com/python/typeshed/blob/695d41f4870aede6e7027cf23bb65b07b59eedc6/stdlib/typing.pyi#L860 for reference

Reply:
And its documented: https://docs.python.org/3/library/collections.html#collections.somenamedtuple._field_defaults